app/src/main/res/layout/trending_option_fragment.xml

```diff
@@ -0,0 +1,145 @@
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/black"
+            android:orientation="vertical"
+            >
+
+            <com.xmartlabs.daydreaming.ui.DiagonalLayoutView
+                android:id="@+id/art_option_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:src="@drawable/art"
+                android:text="@string/art"
+                android:textColor="@color/warm_grey"
+                app:diagonal_angle="7"
+                />
+
+            <com.xmartlabs.daydreaming.ui.DiagonalLayoutView
+                android:id="@+id/first_black_line_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:src="@color/black"
+                app:diagonal_angle="7"
+                />
+
+            <com.xmartlabs.daydreaming.ui.DiagonalLayoutView
+                android:id="@+id/nature_option_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:src="@drawable/nature"
+                android:text="@string/nature"
+                android:textColor="@color/warm_grey"
+                app:diagonal_angle="7"
+                />
+
+            <com.xmartlabs.daydreaming.ui.DiagonalLayoutView
+                android:id="@+id/second_black_line_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:src="@color/black"
+                app:diagonal_angle="7"
+                />
+
+            <com.xmartlabs.daydreaming.ui.DiagonalLayoutView
+                android:id="@+id/travel_option_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:src="@drawable/traveling"
+                android:text="@string/travel"
+                android:textColor="@color/warm_grey"
+                app:diagonal_angle="7"
+                />
+
+            <com.xmartlabs.daydreaming.ui.DiagonalLayoutView
+                android:id="@+id/third_black_line_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:src="@color/black"
+                app:diagonal_angle="7"
+                />
+
+            <com.xmartlabs.daydreaming.ui.DiagonalLayoutView
+                android:id="@+id/animals_option_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:src="@drawable/animals"
+                android:text="@string/animals"
+                android:textColor="@color/warm_grey"
+                app:diagonal_angle="7"
+                />
+
+            <com.xmartlabs.daydreaming.ui.DiagonalLayoutView
```

Comment:
Yeah, i agree. But, i don't mind here taking the 'incremental approach'. So, if we move the borders to the proper file/class/view, then the only thing remaining would be creating the adapter and the view holder.
However, as discussed, the end game should definitely be a recycler view